### PR TITLE
feat(link-blocks): homogenization - INNO-757

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Homogenization of component

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)

### CODE REVIEW

**Review by yhuard — COMMENTED** [review]
I have some concerns about the `wrapper` logic here. I think the template could be simplified...

File: framework/components/ecl-link-blocks/ecl-link-blocks.twig [hunk 1]
```diff
@@ -1,24 +1,38 @@
...
   {% endfor %}
 {% endif %}
 
-{% if wrapper is defined and wrapper == true %}
+{% if wrapper == true %}
 
-<div class="{{ _cssClass }}" {{ _extraAttributes }}>
-  {% if title is defined %}
+<div class="{{ _css_class }}" {{ _extra_attributes }}>
+  {% if title != '' %}
```
Comment (kalinchernev) [inline framework/components/ecl-link-blocks/ecl-link-blocks.twig]: The `if title is not [empty](https://twig.symfony.com/doc/2.x/tests/empty.html)` might be handy.

File: framework/components/ecl-link-blocks/ecl-link-blocks.twig [hunk 2]
```diff
@@ -1,24 +1,38 @@
+{#
+  - "wrapper" (boolean): toggle display of a div wrapper around the component (default: false)
+  - "title" (string): optional title of the link block (default: '')
+  - "links" (array): [
+    "href" (string): target of the link
+    "label" (string): label of the link
+  ]
```
Comment (kalinchernev) [inline framework/components/ecl-link-blocks/ecl-link-blocks.twig]: It's not a problem, but the indentation here takes the eyes a bit.

**Comment by emeryro** [comment]
Wrapper variable has been removed

**Review by yhuard — COMMENTED** [review]
Looks nice! If the tests pass, then we can merge. But latest travis builds failed, so you may have to update some screenshots

**Review by yhuard — CHANGES_REQUESTED** [review]
`dropdowns collapsed dropdowns` screenshots need to be updated

**Review by degliwe — DISMISSED** [review]
refactoring of link-blocks breaks the dropdown (extraClass -> extraClasses)

**Comment by emeryro** [comment]
Dropdown fixed, Travis is happy

**Review by yhuard — APPROVED** [review]

**Pull request merged**